src/renderer/utils/createStorage.ts

```diff
@@ -68,32 +63,35 @@ export function createAppStorage<T>(key: string, defaultValue: T, clusterId?: st
       });
 
     // bind auto-saving data changes to %storage-file.json
-    reaction(() => toJS(storage.data), saveFile, {
-      delay: 250, // lazy, avoid excessive writes to fs
-      equals: comparer.structural, // save only when something really changed
-    });
+    reaction(
+      () => toJS(storage.data),
+      async (state: Record<string, any> = {}) => {
```

Comment:
Was this change necessary? Usually code blocks are refactored to smaller logical chunks, this goes opposite way.